kasa/aestransport.py

```diff
@@ -41,6 +42,14 @@ def _sha1(payload: bytes) -> str:
     return sha1_algo.hexdigest()
 
 
+class AesState(Enum):
```

Comment:
```suggestion
class TransportState(Enum):
```
This way we can re-use this (if needed) later on with other protos.

Reply:
ec7ec9c0ba45602abd8dc3eeae8a28aeb115992d